### PR TITLE
Gives simple humans armor

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request

Give simple humans armor by grabbing the behavior from asteroid mobs and giving them to simple mobs

## Why It's Good For The Game
@Apogee-dev and I have discussed this before, simple humans, even with armor are very weak to low calibers, its "balanced" out with more health, which isn't ideal. By doing this we encourage people to use higher calibers with simplemobs, otherwise they would be reserved for pvp

Seeing as the next gun expansion is exclusively 22 lr, and 22lr is disproportinately good against simplemobs vs actual players, this is better to do now
## Changelog

:cl:
add: Gives simple humans armor!
/:cl:

### CODE REVIEW

**Comment by thgvr** [comment]
Should be fine, this is pretty much how I balanced fauna

**Review by FalloutFalcon — CHANGES_REQUESTED** [review]
i think it would make sense to define the values of amour vests and use them here to ensure they aren't as arbitrary..

**Comment by rye-rice** [comment]
Out of scope, these values were taken from the armor values the sprites the simplemobs use

**Comment by FalloutFalcon** [comment]
> Out of scope, these values were taken from the armor values the sprites the simplemobs use

Me and theubernyan both consider this in scope.

File: code/modules/mob/living/simple_animal/hostile/human/frontiersman.dm [hunk 1]
```diff
@@ -196,13 +198,15 @@
 	name = "Frontiersman Officer"
 	icon_state = "frontiersmanofficer"
 	icon_living = "frontiersmanofficer"
-	maxHealth = 65
-	health = 65
-	rapid = 4
+	maxHealth = 100
+	health = 100
+	rapid = 7
+	rapid_fire_delay = 1
```
Comment (FalloutFalcon) [inline code/modules/mob/living/simple_animal/hostile/human/frontiersman.dm:198]: Undocumented

**Review by FalloutFalcon — CHANGES_REQUESTED** [review]
please define the amour values...
also clarify in the pr that you drop the max health to compensate
and also you could just remove the maxHealth and health from most of these assuming its 100 for the types above it but i wont force you to....

File: code/modules/mob/living/simple_animal/hostile/human/syndicate.dm [hunk 2]
```diff
@@ -79,9 +83,10 @@
 	icon_state = "syndicate_stormtrooper_knife"
 	icon_living = "syndicate_stormtrooper_knife"
 	name = "Ramzi Clique Stormtrooper"
-	maxHealth = 250
-	health = 250
+	maxHealth = 100
+	health = 100
```
Comment (SomeguyManperson) [inline code/modules/mob/living/simple_animal/hostile/human/syndicate.dm]:
```suggestion
```

File: code/modules/mob/living/simple_animal/hostile/human/syndicate.dm [hunk 3]
```diff
@@ -43,8 +45,9 @@
 	icon_state = "syndicate_stormtrooper"
 	icon_living = "syndicate_stormtrooper"
 	name = "Ramzi Clique Assault Trooper"
-	maxHealth = 250
-	health = 250
+	maxHealth = 100
+	health = 100
```
Comment (SomeguyManperson) [inline code/modules/mob/living/simple_animal/hostile/human/syndicate.dm]:
```suggestion
```

File: code/modules/mob/living/simple_animal/hostile/human/syndicate.dm [hunk 4]
```diff
@@ -62,13 +65,14 @@
 	icon_state = "syndicate_space_knife"
 	icon_living = "syndicate_space_knife"
 	name = "Ramzi Clique Commando"
-	maxHealth = 170
-	health = 170
+	maxHealth = 100
+	health = 100
```
Comment (SomeguyManperson) [inline code/modules/mob/living/simple_animal/hostile/human/syndicate.dm]:
```suggestion
```

File: code/modules/mob/living/simple_animal/hostile/human/syndicate.dm [hunk 5]
```diff
@@ -20,19 +20,21 @@
...
 ///////////////Melee////////////
 
 /mob/living/simple_animal/hostile/human/syndicate/space
 	icon_state = "syndicate_space"
 	icon_living = "syndicate_space"
 	name = "Ramzi Clique Commando"
-	maxHealth = 170
-	health = 170
+	maxHealth = 100
+	health = 100
```
Comment (SomeguyManperson) [inline code/modules/mob/living/simple_animal/hostile/human/syndicate.dm]:
```suggestion
```

File: code/modules/mob/living/simple_animal/hostile/human/nanotrasen.dm [hunk 6]
```diff
@@ -55,15 +56,16 @@
...
 /mob/living/simple_animal/hostile/human/nanotrasen/elite
 	name = "Nanotrasen Elite Assault Officer"
 	desc = "Pray for your life, syndicate. Run while you can."
 	icon = 'icons/mob/simple_human.dmi'
 	icon_state = "nanotrasen_ert"
 	icon_living = "nanotrasen_ert"
-	maxHealth = 150
-	health = 150
+	maxHealth = 100
+	health = 100
```
Comment (SomeguyManperson) [inline code/modules/mob/living/simple_animal/hostile/human/nanotrasen.dm]:
```suggestion
```

File: code/modules/mob/living/simple_animal/hostile/human/syndicate.dm [hunk 7]
```diff
@@ -115,13 +120,14 @@
 	icon_state = "syndicate_space_sword"
 	icon_living = "syndicate_space_sword"
 	name = "Ramzi Clique Commando"
-	maxHealth = 170
-	health = 170
+	maxHealth = 100
+	health = 100
```
Comment (SomeguyManperson) [inline code/modules/mob/living/simple_animal/hostile/human/syndicate.dm]:
```suggestion
```

File: code/modules/mob/living/simple_animal/hostile/human/syndicate.dm [hunk 8]
```diff
@@ -137,9 +143,10 @@
 	icon_state = "syndicate_stormtrooper_sword"
 	icon_living = "syndicate_stormtrooper_sword"
 	name = "Ramzi Clique Stormtrooper"
-	maxHealth = 250
-	health = 250
+	maxHealth = 100
+	health = 100
```
Comment (SomeguyManperson) [inline code/modules/mob/living/simple_animal/hostile/human/syndicate.dm]:
```suggestion
```

File: code/modules/mob/living/simple_animal/hostile/human/syndicate.dm [hunk 9]
```diff
@@ -163,12 +170,13 @@
 	icon_state = "syndicate_space_pistol"
 	icon_living = "syndicate_space_pistol"
 	name = "Ramzi Clique Commando"
-	maxHealth = 170
-	health = 170
+	maxHealth = 100
+	health = 100
```
Comment (SomeguyManperson) [inline code/modules/mob/living/simple_animal/hostile/human/syndicate.dm]:
```suggestion
```

File: code/modules/mob/living/simple_animal/hostile/human/syndicate.dm [hunk 10]
```diff
@@ -179,8 +187,9 @@
 	icon_state = "syndicate_stormtrooper_pistol"
 	icon_living = "syndicate_stormtrooper_pistol"
 	name = "Ramzi Clique Stormtrooper"
-	maxHealth = 250
-	health = 250
+	maxHealth = 100
+	health = 100
```
Comment (SomeguyManperson) [inline code/modules/mob/living/simple_animal/hostile/human/syndicate.dm]:
```suggestion
```

File: code/modules/mob/living/simple_animal/hostile/human/syndicate.dm [hunk 11]
```diff
@@ -193,12 +202,13 @@
 	icon_state = "syndicate_space_smg"
 	icon_living = "syndicate_space_smg"
 	name = "Ramzi Clique Commando"
-	maxHealth = 170
-	health = 170
+	maxHealth = 100
+	health = 100
```
Comment (SomeguyManperson) [inline code/modules/mob/living/simple_animal/hostile/human/syndicate.dm]:
```suggestion
```

File: code/modules/mob/living/simple_animal/hostile/human/syndicate.dm [hunk 12]
```diff
@@ -224,12 +235,13 @@
 	icon_state = "syndicate_space_shotgun"
 	icon_living = "syndicate_space_shotgun"
 	name = "Ramzi Clique Commando"
-	maxHealth = 170
-	health = 170
+	maxHealth = 100
+	health = 100
```
Comment (SomeguyManperson) [inline code/modules/mob/living/simple_animal/hostile/human/syndicate.dm]:
```suggestion
```

File: code/modules/mob/living/simple_animal/hostile/human/syndicate.dm [hunk 13]
```diff
@@ -240,8 +252,9 @@
 	icon_state = "syndicate_stormtrooper_shotgun"
 	icon_living = "syndicate_stormtrooper_shotgun"
 	name = "Ramzi Clique Stormtrooper"
-	maxHealth = 250
-	health = 250
+	maxHealth = 100
+	health = 100
```
Comment (SomeguyManperson) [inline code/modules/mob/living/simple_animal/hostile/human/syndicate.dm]:
```suggestion
```

File: code/modules/mob/living/simple_animal/hostile/human/frontiersman.dm [hunk 14]
```diff
@@ -196,13 +198,15 @@
 	name = "Frontiersman Officer"
 	icon_state = "frontiersmanofficer"
 	icon_living = "frontiersmanofficer"
-	maxHealth = 65
-	health = 65
-	rapid = 4
+	maxHealth = 100
+	health = 100
```
Comment (SomeguyManperson) [inline code/modules/mob/living/simple_animal/hostile/human/frontiersman.dm]:
```suggestion
```

File: code/modules/mob/living/simple_animal/hostile/human/frontiersman.dm [hunk 15]
```diff
@@ -166,8 +167,9 @@
 	icon_state = "frontiersmanrangedminigun"
 	icon_living = "frontiersmanrangedminigun"
 	projectilesound = 'sound/weapons/laser4.ogg'
-	maxHealth = 260
-	health = 260
+	maxHealth = 100
+	health = 100
```
Comment (SomeguyManperson) [inline code/modules/mob/living/simple_animal/hostile/human/frontiersman.dm]:
```suggestion
```

File: code/modules/mob/living/simple_animal/hostile/human/frontiersman.dm [hunk 16]
```diff
@@ -82,8 +82,9 @@
 /mob/living/simple_animal/hostile/human/frontier/ranged/trooper
 	icon_state = "frontiersmanrangedelite"
 	icon_living = "frontiersmanrangedelite"
-	maxHealth = 170
-	health = 170
+	maxHealth = 100
+	health = 100
```
Comment (SomeguyManperson) [inline code/modules/mob/living/simple_animal/hostile/human/frontiersman.dm]:
```suggestion
```

**Review by SomeguyManperson — COMMENTED** [review]
All of these are inherited from the default hostile human type so they can be removed

**Comment by rye-rice** [comment]
@Apogee-dev , i delegate YOU

**Pull request closed without merging**